### DISCUSSION TITLE
Tahoma2d copy and paste

### DISCUSSION THREAD

**Discussion by Dixxi0509** [tahoma2d/tahoma2d#1164] [General]
Hey I have a question whenever I copy and paste my animation and try to edit it that effects the animation from before how do I make it stop?

**Comment by collossuspilot55** [comment]
Hi,

Right click the frame you want to copy. Then go to the frame you want the copy to be on, right click it, go to the menu labeled "Paste Special", and finally click "Paste as a Copy".

**Comment by Dixxi0509** [comment]
Would that work?

**Reply by collossuspilot55** [reply]
Yep

**Reply by collossuspilot55** [reply]
You just have to do that every time you want to copy a frame, which isn't too bad once you get used to it.

**Comment by manongjohn** [comment]
Can use keyboard shortcut 'D' to create a duplicated image of the current image you are on.

Works provided there is an empty cell next to the one you want to duplicate.

**Comment by Dixxi0509** [comment]
Thanks

**Comment by Dixxi0509** [comment]
Hey it didn't work
![IMG_20230527_225242_HDR](https://github.com/tahoma2d/tahoma2d/assets/91355354/cdacbfb8-bb1c-4738-bbf4-3b480ff122e4)
![IMG_20230527_225237_HDR](https://github.com/tahoma2d/tahoma2d/assets/91355354/ebc5e520-2fd4-4459-a1ed-b1b634245e9c)

**Reply by collossuspilot55** [reply]
You have to copy the frame first, also like John said, just hitting D while on the frame you want works too.

**Reply by manongjohn** [reply]
What didnt work? Btw, use lowercase 'd'.

Note: if you are using Implicit Mode, using shortcut 'd' on an implicit cell will convert it to  Duplicate.

**Comment by Dixxi0509** [comment]
The special paste
